Library/Formula/cv.rb

```diff
@@ -0,0 +1,20 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/Xfennec/cv"
+  url "https://github.com/Xfennec/cv/archive/v0.6.tar.gz"
+  sha256 "4291ab2f0c507df9eb694bf956ed65dc12bae3c492b795235142b5b87e2e2239"
+  head "https://github.com/Xfennec/cv.git"
+
+  patch do
+    url "https://patch-diff.githubusercontent.com/raw/Xfennec/cv/pull/19.diff"
+    sha256 "3b3b702ad6d51131ef235ebd3544709d28d7f0d39a0fabe833f4909202819b76"
+  end
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system "cv", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!